packages/backend/src/lib/Ethereum/Ethereum.test.ts

```diff
@@ -0,0 +1,30 @@
+import { Ethereum } from './Ethereum';
+import { EthereumConfigMintNFT } from './EthereumConfig/EthereumConfigMintNFT';
+
+describe('Ethereum', () => {
+  it.skip('should estimate gas', async () => {
+    const ethereum = new Ethereum();
+
+    const contract_adress = '0xB924387088C7c2961f3049002D9474b5F308F8c4';
+    const priv_key_NFT_transmitter =
+      '36b802d163dea869795fa3ebd5f671743038a4fc4f9d8b0e3538de4fc1a1d8e8';
+    const pub_key_NFT_receiver = '0xA995ECea55f0739d07B8F3eEF8153E58C1e838C8';
```

Comment:
i guess this test won't even work without running ganache and changing these parameters every time?

Reply:
yes exactly. As the name of the command indicates, this functionality is just here, so we can quickly test our minting function without having to go through the whole start minting input process.